internal/versions/versions_test.go

```diff
@@ -75,6 +75,24 @@ var defaultOsImages = models.OsImages{
 }
 
 var defaultReleaseImages = models.ReleaseImages{
+	// Two images below represent a scenario when the same OpenShift Version (as reported by the
+	// CVO) is provided by more than a single release image. This is a scenario when for the same
+	// OCP version we have single-arch and multi-arch image. This happens because starting from
```

Comment:
Why would we do this? Shouldn't we only provide the multi-arch image in this case? Or do some multi-arch images not include all the possible architectures?

Reply:
No, because multi-arch is not supported. For 4.12 we offer both multi-arch and single-arch. But it must be always the user's conscious decision to pick the multi-arch one